wagtail/admin/views/generic/base.py

```diff
@@ -251,7 +251,11 @@ def active_filters(self):
 
         for field_name in self.filters.form.changed_data:
             filter_def = self.filters.filters[field_name]
-            value = self.filters.form.cleaned_data[field_name]
+            try:
+                value = self.filters.form.cleaned_data[field_name]
+            except KeyError:
+                continue  # invalid filter value
```

Comment:
Thanks! I was working on this for a separate PR, but I'm happy to get this in here. I'll add the test as I merge.